packages/fiori/src/ViewSettingsDialog.hbs

```diff
@@ -54,7 +54,7 @@
 			{{#if isModeSort}}
 				<div class="ui5-vsd-sort">
 					<ui5-list
-						mode="SingleSelectBegin"
+						selection-mode="SingleSelectBegin"
```

Comment:
I believe SingleSelectBegin should be SingleStart.

Same for the rest of the occurrences.